csv_to_openfisca.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+import pandas as pds
+import numpy as np
+
+# from openfisca_core.simulations import Simulation
+from openfisca_core.simulation_builder import SimulationBuilder, Simulation
+from openfisca_country_template import CountryTaxBenefitSystem
+
+tax_benefit_system = CountryTaxBenefitSystem()
+
+def set_members_entity_id(entity, entity_id_array):
+  assert entity.ids is not None  # We needs the ids to be set to make sure we match households.
+  ids, inverse = entity.members_entity_id = np.unique(entity_id_array, return_inverse = True)
```

Comment:
`inverse ` would be what we want, if in the household dataframe ids were always sorted. As it's not the case, we have an extra mapping processing.